src/NuGet.Core/NuGet.Protocol.Core.Types/Model/PackageSearchMetadataBuilder.cs

```diff
@@ -123,5 +126,10 @@ public static IPackageSearchMetadata WithVersions(this IPackageSearchMetadata me
                 .WithVersions(AsyncLazy.New(valueFactory))
                 .Build();
         }
+
+        public static string GetFullName(this IPackageSearchMetadata iPackageSearchMetadata)
```

Comment:
Old code was using ToString on SemanticVersion so I kept it same, but maybe there's an argument that this needs changed?